README.md

```diff
@@ -61,6 +61,12 @@ The official constitution of the UNSW Security Society.
       3.6.2 Cease to be a member of the club;
       3.6.3 Cease to be a UNSW student;
       3.6.4 Are absent from any three (3) consecutive meetings of the club without apology or leave; or
```

Comment:
It's probably my ignorance of legal terminology, but what exactly constitutes 'apology or leave'?
I feel as if this can be interpreted in several ways and should be explicitly defined.